File: affinity/sources/affinity.5.c
/*
* @@name:	affinity.5
* @@type:	C
* @@operation:	compile
* @@expect:	success
* @@version:	omp_5.1
*/
void work();
int main()
{
#pragma omp parallel proc_bind(primary) num_threads(4)
   {
      work();
   }
   return 0;
}

File: affinity/sources/affinity.5.f
! @@name:	affinity.5
! @@type:	F-fixed
! @@operation:	compile
! @@expect:	success
! @@version:	omp_5.1
      PROGRAM EXAMPLE
!$OMP PARALLEL PROC_BIND(primary) NUM_THREADS(4)
      CALL WORK()
!$OMP END PARALLEL
      END PROGRAM EXAMPLE
